caps/types.go

```diff
@@ -90,6 +111,21 @@ func (t *TestType) Sanitize(c *Capability) error {
 	return nil
 }
 
+// SecuritySnippet for test capability type.
+// Consumers don't gain any extra permissions.
+func (t *TestType) SecuritySnippet(c *Capability, securitySystem string) (string, error) {
+	switch securitySystem {
+	case SecurityApparmor:
+		fallthrough
```

Comment:
Seems wrong to fallthrough here. It's trivial to return an empty snippet, and if any of these are actually implemented the actual implementation will definitely not be common.

On another angle, what security system would be unsupported for the test type, considering it doesn't require anything at all from the system to implement the capability?  Might be worth returning an empty snippet in all cases, and maybe allow a callback similar to Sanitize if that turns out to be useful?